orderBy.js

```diff
@@ -41,7 +41,14 @@ function orderBy(collection, iteratees, orders) {
   }
   if (!Array.isArray(iteratees)) {
     iteratees = iteratees == null ? [] : [iteratees]
+    iteratees = iteratees.map((iteratee) => {
+      if (Array.isArray(iteratee)) {
+        return iteratee.join('.')
+      }
```

Comment:
We don't want to use `join()` here as array paths can capture things not possible or more complex in dot form.

Reply:
Could you give me an example of such a case, I'll try improving this PR :)